compiler/CMakeLists.txt

```diff
@@ -63,14 +63,20 @@ if(THEROCK_ENABLE_COMPILER)
   # runtime.
   ##############################################################################
 
+  if(WIN32)
+    set(_link_dylib "OFF")
+  else()
+    set(_link_dylib "ON")
+  endif()
+
   therock_cmake_subproject_declare(amd-comgr
     EXTERNAL_SOURCE_DIR "amd-llvm/amd/comgr"
     BINARY_DIR "amd-comgr"
     BACKGROUND_BUILD
     CMAKE_ARGS
       # TODO: Currently unstable. Enable in >6.4.
       -DCOMGR_DISABLE_SPIRV=ON
-      -DLLVM_LINK_LLVM_DYLIB=ON
+      -DLLVM_LINK_LLVM_DYLIB=${_link_dylib}
```

Comment:
You can just take this out and put it all properly conditioned in the pre_hook.

Reply:
(I think this was just leftover from before I moved all of the logic there)

Reply:
How's this? Want
```cmake
    CMAKE_ARGS
      # TODO: Currently unstable. Enable in >6.4.
      -DCOMGR_DISABLE_SPIRV=ON
```

moved too?